src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -233,15 +229,16 @@ g.test('GPUBlendComponent')
     const renderPass = commandEncoder.beginRenderPass({
       colorAttachments: [
         {
-          attachment: renderTarget.createView(),
+          view: renderTarget.createView(),
           loadValue: dstColor,
+          storeOp: 'store',
         },
       ],
     });
     renderPass.setPipeline(pipeline);
-    if (blendColor) {
+    if (blendConstant) {
       /* eslint-disable-next-line deprecation/deprecation */
```

Comment:
Can this comment be removed now?

Reply:
Oops, yes. I thought I got all these, evidently not.